### PR TITLE
add doc of matrixone-function-list

### PR DESCRIPTION
**Pull request by yangj1211**
## What type of PR is this?

- [ ] Enhancement
- [ ] Displaying
- [ ] Typo
- [x] Doc Request

## Which issue(s) this PR fixes:

issue #908

### CODE REVIEW

**Comment by yangj1211** [comment]
@dengn need to review

File: docs/MatrixOne/Overview/matrixone-function-list.md
Comment (dengn) [inline docs/MatrixOne/Overview/matrixone-function-list.md:1]: 把这个放到函数列表里面，然后在上面加一个 “函数总表”页面。

image

**Review by dengn — CHANGES_REQUESTED** [review]

**Comment by yangj1211** [comment]
@dengn need to review
已更改至Reference/Functions-and-Operators目录下

**Review by dengn — APPROVED** [review]

**Comment by dengn** [comment]
Main里面也要更新一个函数列表

**Pull request merged**